test/access-token.test.js

```diff
@@ -15,6 +15,10 @@ var ds = loopback.createDataSource({ connector: loopback.Memory });
 Token.attachTo(ds);
 var ACL = loopback.ACL;
 
+beforeEach(function() {
+  this.app = app = loopback();
```

Comment:
Please define a describe-local `var app` and move the hook into the top-level describe block, so that we don't pollute global namespace and don't run this hook for tests that don't need it.

```js
describe('loopback.token(options)', function() {
  var app;
  beforeEach(function() {
    this.app = app = loopback();
  });

  // ...
});
```